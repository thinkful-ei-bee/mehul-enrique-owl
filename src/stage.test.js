import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './stage';
import renderer from 'react-test-renderer';


it('renders Stage without crasing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <Stage
    name='name'
    avatar='alt-text'
  />,
    div,
  );
  ReactDOM.unmountComponentAtNode(div);
 
});

it('renders the Stage as expected', () => {
  const tree = renderer
    .create(<Stage name="name" avatar='alt-text'/>)
    .toJSON();
  expect(tree).toMatchSnapshot();  
});