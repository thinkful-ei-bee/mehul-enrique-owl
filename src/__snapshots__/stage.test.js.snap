// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Stage as expected 1`] = `
<li
  className="stage-obj"
>
  <p>
    name
     
  </p>
  <img
    alt="avatar-text"
    className="avatar-stage"
    src="alt-text"
  />
</li>
`;
